components/form/FAutocompleteInput.vue

```diff
@@ -44,45 +47,44 @@ FField.FAutocompleteInput(
         hide-error-icon
         hide-hint
         @focus="e => handleFocus(e, openMenu)"
-        @blur="handleBlur"
-        @change="handleChange"
+        @blur="e => handleBlur(e, closeMenu)"
+        @change="e => emit('change', e)"
         @input="handleInput"
```

Comment:
More simply what you can do here, instead of passing the function as, a callback:
```pug
        @focus="openMenu(); handleFocus($event)"
        @blur="closeMenu(); handleBlur($event)"
        @change="emit('change', $event)"
```